packages/jellyfish-api-core/__tests__/category/rawtx/decodeScript.test.ts

```diff
@@ -0,0 +1,40 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+
+describe('Decodescript', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForWalletCoinbaseMaturity()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  async function createToken (symbol: string): Promise<string> {
```

Comment:
Use the `testing` module in `jellyfish-testing`; I think some `jellyfish-api-core/__tests__` uses them; you try to find one for reference and use it.

Reply:
Would you be referring to this?

`{ TestingGroup } from '@defichain/jellyfish-testing'`

Reply:
Just `Testing`

```ts
const testing = Testing.create(new MasterNodeRegTestContainer())
```


`TestingGroup` is when you want to use multiple containers. `Testing` is singular.